flask_login/login_manager.py

```diff
@@ -288,6 +288,18 @@ def needs_refresh(self):
         return redirect(redirect_url)
 
     def reload_user(self, user=None):
+        '''
```

Comment:
Adding a docstring is good, but it should follow the same format as other docstrings with a one sentence description, and more info following.